src/pmi/pmi_wrapper.c

```diff
@@ -47,6 +52,7 @@ void lcm_pm_initialize()
           word);
     if (LCM_PM_ops.check_availability()) {
```

Comment:
There's still an issue where e.g. the `LCI_PM_BACKEND` env var is set to `""` and so nothing gets selected and then you try to call `LCM_PM_ops.check_availability()` even though it's `NULL`. That's the problem we need to fix.

Reply:
Do you mean set the env var to an empty string?

Reply:
Then `getenv` should return NULL, right?

Reply:
Also `strtok_r` will return NULL if it's an empty string.